config/nvim/lua/user/types/wezterm/types.lua

```diff
@@ -1083,6 +1083,7 @@ function KeyAssignment.SwitchWorkspaceRelative(direction) end
 ---@field key_tables table<string, _.wezterm.KeyBinding[]>
 ---@field keys _.wezterm.KeyBinding[]
 ---@field launch_menu _.wezterm.SpawnCommand[]
+---@field leader _.wezterm.KeyBinding[]
```

Comment:
This property has type LeaderKey (which isn't in these types yet) rather than KeyBinding[]. From looking at the [docs](https://wezfurlong.org/wezterm/config/keys.html?h=leader#leader-key) and the source, that type should have key, mods, and timeout_milliseconds properties.